npm/private/lifecycle/lifecycle-hooks.js

```diff
@@ -124,6 +129,19 @@ async function runLifecycleHooks(opts, hooks) {
     }
 }
 
+// Remove the indeterministic Makefile from the resulting directory.
+// Can be removed when https://github.com/nodejs/gyp-next/pull/293
+// has sufficiently propagated.
```

Comment:
When will it be "sufficiently propagated" and how can we tell?

Reply:
Can you add that version info to the comment? Just so some day when scanning this if someone notices node-gyp@11.3.0 and knows it's no longer supported then we can drop it...